liquidity_pool/Cargo.toml

```diff
@@ -9,21 +9,18 @@ crate-type = ["cdylib", "rlib"]
 
 [features]
 token-wasm = []
-testutils = ["soroban-sdk/testutils", "soroban-token-contract/testutils", "dep:ed25519-dalek", "dep:sha2", "dep:stellar-xdr"]
+testutils = ["soroban-sdk/testutils", "soroban-auth/testutils", "dep:ed25519-dalek", "dep:sha2", "dep:stellar-xdr"]
 
 [dependencies]
 soroban-sdk = "0.0.4"
 soroban-auth = "0.0.4"
+soroban-token-spec = "0.0.4"
```

Comment:
The token spec shouldn't be a dependency. We could make it a dependency only for non-wasm builds, or for dev/tests only, but if it is included here the token contract will be reexported in the liquidity pool contract.

Reply:
Good point. Removed references to the token spec.